pkg/auth/rbac.go

```diff
@@ -40,15 +40,16 @@ const (
 	PermissionDeleteEnvironmentApplication = "DeleteEnvironmentApplication"
 	PermissionDeployReleaseTrain           = "DeployReleaseTrain"
 	// The default permission template.
-	PermissionTemplate = "%s,%s,%s:%s,%s,allow"
+	PermissionTemplate = "p,role:%s,%s,%s:%s,%s,allow"
```

Comment:
please add the docker-compose changes you mention in the commit message to the docker-compose earthly file.
(see this internal document https://revolution.dev/app/-JqFGExX46gs9mH7vxR5/-M37cZx7XYoOvQ5Mmcir/DOCUMENT/-NvBzabmtucDxkHg3hI_/ )